docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -27,10 +27,10 @@ Once your tax provider configuration is ready, we'll let you know through email.
 | Tax Provider Details | Required / Optional | Value(s) | Description | Example |
 | :--- | :--- | :--- | :--- | :--- |
 | App ID | Required | Integer | Tells us which tax provider configuration to use after the app is installed. | `123456` |
-| Tax provider name | Required | String | Displayed in the active MSF-enabled BigCommerce control panel (e.g. **Settings > Setup > Tax > Add tax service**). | `Sample Tax` |
+| Tax provider name | Required | String | Displayed in the BigCommerce control panel (e.g. **Settings > Setup > Tax > Add tax service**). | `Sample Tax` |
 | Tax provider type | Required | Production, Sandbox | Hierarchy of tax provider configurations, Production is primary and Sandboxes are secondary, see [sandbox tax provider configuration](#sandbox-tax-provider-configuration) for more information. | `Production` |
 | Partner support email | Required | Email | Used by BigCommerce to contact tax provider, in the case we need to forward a merchant support request. | `support@sampletax.example.com` |
-| Links displayed in an active MSF-enabled control panel | Optional | URL, Title, Description | Link(s) displayed to merchants when they navigate to **Settings > Setup > Tax > _TaxProviderName_**. One or multiple links supported. | `support.sampletax.com`, `Sample Tax Support`, `The Sample Tax Support website.` |
+| Links displayed in a control panel | Optional | URL, Title, Description | Link(s) displayed to merchants when they navigate to **Settings > Setup > Tax > _TaxProviderName_**. One or multiple links supported. | `support.sampletax.com`, `Sample Tax Support`, `The Sample Tax Support website.` |
```

Comment:
I'm thinking instead of saying a control panel or a BigCommerce control panel we just say the control panel or the BigCommerce control panel. Any thoughts on this?

Reply:
Good point, just changed my doc

Reply:
This was written like this because we needed to differentiate between when MSF was active vs. inactive. The control panel changes. The menus are different.

Reply:
@bc-traciporter The menus (for what is in the doc) are now the same for MSF and non-MSF stores. I just checked.

Reply:
Oh nice, that's awesome.

Reply:
@bc-traciporter The tax settings UI though did not change based on MSF vs non-MSF, so a bit confused why this extra detail was added.

For example, the links mentioned here would still display exactly the same regardless.